web/models/package_metadata.ex

```diff
@@ -0,0 +1,26 @@
+defmodule HexWeb.PackageMetadata do
+  use HexWeb.Web, :model
+
+  embedded_schema do
+    field :description, :string
+    field :licenses, {:array, :string}
+    field :links, :map
+    field :maintainers, {:array, :string}
+    field :contributors, {:array, :string}
+  end
+
+  @required_fields ~w(description)
+  @optional_fields ~w(licenses links maintainers)
+
+  def changeset(meta, params \\ :empty) do
+    cast(meta, params, @required_fields, @optional_fields)
+    |> validate_presence(:description)
```

Comment:
This will be `validate_required` once we move to Ecto 2.0. Maybe we should leave a TODO note?